pm2.json

```diff
@@ -0,0 +1,10 @@
+{
+  "apps": [
+    {
+      "name": "logsServer",
+      "script": "lib/indexApi.js",
+      "error_file": "logs/error.log",
```

Comment:
I didn't catch this in other PRs, but I think we should not include log files in the repo directory. Any time an server admin does `git reset --hard` in the repo we will lose our log files. Or, maybe add the dir to `.gitignore`.